pegasus/sites.v3/code.org/public/teacher-dashboard/section_manage.haml

```diff
@@ -4,6 +4,55 @@ layout: none
 theme: none
 content-type: text/ng-template
 ---
+-# TODO: Move this into a separate partial
```

Comment:
I wanted to do this and spent a little bit of time digging into it back then, but wasn't sure if it was possible. Is it?